pages/resources/guild.mdx

```diff
@@ -1357,6 +1357,51 @@ Defines the criteria used to satisfy Onboarding constraints that are required fo
 }
 ```
 
+### Premium Subscription Object
+
+A premium subscription represents a member's premium subscription ([boost](https://support.discord.com/hc/en-us/sections/360007875211-Server-Boosting)) in a guild.
+
+###### Premium Subscription Structure
+
+| Field             | Type                                                              | Description                                                         |
+| ----------------- | ----------------------------------------------------------------- | ------------------------------------------------------------------- |
+| id                | snowflake                                                         | The ID of the subscription                                          |
+| guild_id          | snowflake                                                         | The ID of the guild this subscription is for                        |
+| user_id           | snowflake                                                         | The ID of the user this subscription is for                         |
+| ended             | boolean                                                           | If this premium subscription has ended                              |
+| pause_ends_at ^1^ | ?ISO8601 timestamp                                                | The timestamp for when this user's premium subscription will expire |
+| user              | partial [user](/resources/user#user-object) object                | The user this subscription is for                                   |
+
+^1^ This field will be null in most cases, but it may be set to a timestamp in cases where the user's premium subscription is expiring.
+
+###### Example Premium Subscription
+
+```json
+{
+  "id": "603376323175383060",
+  "user_id": "852892297661906993",
```

Comment:
nitpick but the user id can't be older than the sub id :p 
feel free to use your own user